parser/fsimporter.go

```diff
@@ -575,7 +575,7 @@ func (fs *FSImporter) buildFQDNBeacons(hostnameMap map[string]*hostname.Input, m
 			}
 
 			// send uconns to beacon analysis
```

Comment:
I know this comment is from before, but it needs updated to read "hostMap" instead of uconns